### DISCUSSION TITLE
CRediT

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#200] [General]
Hi everyone! As @Aariq mentions in #199, we can identify how we contributed to the manuscript using the CRediT system and including this in our manuscript.  Copy and past all the different author roles below, and write your initials next to the aspects you contributed to.  Check out my post below as an example!

Conceptualization
Data curation
Formal Analysis
Funding acquisition
Investigation
Methodology
Project administration

Resources
Software
Supervision
Validation
Visualization
Writing – original draft
Writing – review & editing

**Comment by wavingtowaves** [comment]
Conceptualization: RCO
Data curation
Formal Analysis
Funding acquisition
Investigation: RCO, RCO
Methodology: RCO
Project administration: RCO

Resources
Software
Supervision
Validation
Visualization: RCO
Writing – original draft: RCO
Writing – review & editing: RCO

**Comment by Aariq** [comment]
The [CRediT](https://credit.niso.org/) definition for "investigation" is 
>Conducting a research and investigation process, specifically performing the experiments, or data/evidence collection.

So in the context of this manuscript, I think that means anyone who either contributed their own ideas or synthesized ideas from other sources did "evidence collection" and should get credit for investigation.

Conceptualization
Data curation
Formal Analysis
Funding acquisition
Investigation: ERS
Methodology
Project administration

Resources
Software: ERS
Supervision
Validation
Visualization: ERS
Writing – original draft: ERS
Writing – review & editing: ERS

**Reply by wavingtowaves** [reply]
Thanks @Aariq. I've edited my CRediT accordingly.

**Comment by Aguncan** [comment]
Conceptualization 
Data curation
Formal Analysis
Funding acquisition
Investigation AG
Methodology
Project administration: 

Resources
Software AG
Supervision
Validation
Visualization
Writing – original draft: AG
Writing – review & editing: AG

**Reply by Aguncan** [reply]
> 

I hope I did understand them correctly, If I have made a mistake, please inform me and then I will correct it.

**Reply by wavingtowaves** [reply]
That's perfect, thanks @Aguncan !

**Comment by shafieisabets** [comment]
Conceptualization
Data curation
Formal Analysis
Funding acquisition
Investigation SSHS
Methodology
Project administration

Resources
Software SSHS
Supervision
Validation
Visualization
Writing – original draft SSHS
Writing – review & editing SSHS

**Reply by shafieisabets** [reply]
> Conceptualization Data curation Formal Analysis Funding acquisition Investigation SSHS Methodology Project administration
> 
> Resources Software SSHS Supervision Validation Visualization Writing – original draft SSHS Writing – review & editing SSHS

Please correct me if I did it not appropriately.
Kind regards
Saeed
